### PR TITLE
Add new project, edit previous.

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

File: data/projects/generic_bears.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+The task is to fix as many issues with these bears and make them as usable
+as possible.
+
+### Milestones
+
+##### GSoC 2017 Coding Starts
+
+ * All issues are disccused, and a general idea behind the algorithm of
+ most generic bears is developed.
+ * New changes to these algorithms(if required) are proposed.
```
Comment (sils) [inline data/projects/generic_bears.md]: missing space

File: data/projects/generic_bears.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+reliable, AnnotationBear needs to give a better more detailed API, 
+IndentationBear looks cool but is not usable yet. These bears can be found 
+[here](https://github.com/coala/coala-bears/tree/master/bears/general).
+
+The task is to fix as many issues with these bears and make them as usable
+as possible.
+
+### Milestones
+
+##### GSoC 2017 Coding Starts
```
Comment (sils) [inline data/projects/generic_bears.md]: those goals aren't concise enough and allow the student to not be around for the bonding phase largely, we should clarify the goals and make them measurable.

File: data/projects/generic_bears.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+ * All issues are disccused, and a general idea behind the algorithm of
+ most generic bears is developed.
+ * New changes to these algorithms(if required) are proposed.
+
+##### GSoC 2017 Midterm
+
+ * A given set of issues defined in the proposal are resolved.
```
Comment (sils) [inline data/projects/generic_bears.md]: I'd say "Half of the choosen issues" or so

File: data/projects/generic_bears.md (hunk 2 above)
Comment (sils) [inline data/projects/generic_bears.md]: maybe sonthing like "the student has identified at least # important issues" or so?

**Comment by sils** [comment]
https://github.com/coala/projects/pull/153/commits/4c4632e538b6b1e08e35fcb09195d30d469131da is ready

File: data/projects/generic_bears.md (hunk 2 above)
Comment (abhsag24) [inline data/projects/generic_bears.md]: sure

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/153/commits/a86097c6cd5520e382ebaa54b92e70415990ee4f

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/153/commits/f7245e262356b974f3bfa2662a81b3cf6d817443

**Comment by sils** [comment]
@rultor merge

**Pull request merged**